src/embed/http.rs

```diff
@@ -1,96 +1,234 @@
 use std::convert::TryFrom;
```

Comment:
It's great that in Replicache http is required only at this extremely high level. In Noms it is required very low down in the stack.